api/envoy/data/core/v2alpha/local_reply_body.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package envoy.data.core.v2alpha;
+
+option java_outer_classname = "LocalReplyBodyProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.data.core.v2alpha";
+
+// [#protodoc-title:  core data]
```

Comment:
Superfluous spacing? And do you think `core data` here the correct title for this?